### PR TITLE
Tags functionality in Zambia

### PR DESCRIPTION
**Pull request by olszowka**
Rename "Publication Characteristics" (PubChars) to Tags.
Add tags to more pages in Zambia.

### CODE REVIEW

**Comment by olszowka** [comment]
This functionality can be viewed on zambia.dreamhosters.com (the general demo).

**Comment by LVerhulst4321** [comment]
How do you log into the general demo at zambia.dreamhosters.com again? I can't find the details.

**Comment by olszowka** [comment]
User id’s are 1 or 2. Password is demo.

  —PeterO

File: webpages/reports/featuredsessions.php [hunk 1]
```diff
@@ -21,9 +21,9 @@
     WHERE EXISTS ( SELECT
                            sessionid
                        FROM
-                           SessionHasPubChar
+                           SessionHasTag
                        WHERE
-                               pubcharid = 5 /* Featured */
+                               tagid = 5 /* Featured */
```
Comment (LVerhulst4321) [inline webpages/reports/featuredsessions.php]: Tag ID won't always be 5 = Featured unless this in the initial table.

**Pull request merged**